markdown/preface.markdown

```diff
@@ -231,6 +225,7 @@ print(2 * β)
 
 The Python core team decided, rightly, that it was worth supporting characters from all languages as first-class citizens in Python code.
 This is especially true now, when most new coders are from non-English-speaking countries.
+In most cases we wouldn't recommend using Unicode variable names as this may go badly in some text editors and will make the job of people translating your code to Python 2 even harder.
```

Comment:
The "nevertheless" in the next paragraph is now awkward. If we make this sentence a bit more nuanced, the flow can be preserved: "For the sake of interoperability, we still recommend using English characters in most code, but this capability can come in handy, for example, in math-heavy Jupyter notebooks."